cypress/e2e/ScraperLiteEdition/APIInfo.cy.js

```diff
@@ -0,0 +1,97 @@
+import { SCRAPER_BASE_URL } from '../../resources/data';
+
+describe('Scraper(Lite Edition) APIs Info page', () => {
+    before(() => {
+        cy.visit(SCRAPER_BASE_URL);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to check for the "API Info" tab is present', () => {
+        cy.componentVisiblityCheck('#details-tab > .d-md-flex > span');
+    });
+
+    it('3. Test to click on the "API Info" tab', () => {
+        cy.get('#details-tab > .d-md-flex > span').click();
+    });
+
+    it('4. Test to click "Adanced Scraper API" link', () => {
+        cy.get('article > :nth-child(4) > a').click();
+        cy.document().should((doc) => {
+            expect(doc.readyState).to.equal('complete');
+        });
+        cy.assertPathname('/marketplace/adv_scraper-api');
+        cy.checkTextVisibility('Advanced Scraper API');
+        cy.go('back');
+        cy.get('#details-tab > .d-md-flex > span').click();
+    });
+
```

Comment:
Write the test case to check the ul in api info page
![image](https://github.com/user-attachments/assets/b2322e0a-1046-454a-a063-02b83957fce3)

Reply:
Added the Ul list.